Makefile

```diff
@@ -15,7 +15,7 @@
 #
 
 BUILD          ?= $(shell git rev-parse --short HEAD)
-BUILD_CODENAME  = dgraph
+BUILD_CODENAME ?= dgraph
```

Comment:
`BUILD_CODENAME` also defined in the `dgraph/Makefile`. Which one takes precedence?

Reply:
We define `BUILD_CODENAME ?= dgraph` in the nested makefile, which means that, if `BUILD_CODENAME` is defined at the root level, it will take precedence and be inherited at the lower level.  I've also confirmed this locally just to be sure.

Reply:
Thanks!